doc/sphinx/Makefile.am

```diff
@@ -94,18 +93,6 @@ DEPS_Pacemaker_Python_API 	= ../../python
 
 if BUILD_SPHINX_DOCS
 
-INKSCAPE_CMD	= $(INKSCAPE) --export-dpi=90 -C
-
-# Pattern rule to generate PNGs from SVGs
-# (--export-png works with Inkscape <1.0, --export-filename with >=1.0;
-# create the destination directory in case this is a VPATH build)
-%.png: %.svg
-	$(AM_V_at)-$(MKDIR_P) "$(shell dirname "$@")"
-	$(AM_V_GEN) {							\
-		$(INKSCAPE_CMD) --export-png="$@" "$<" 2>/dev/null	\
-		|| $(INKSCAPE_CMD) --export-filename="$@" "$<";		\
-	} $(PCMK_quiet)
-
```

Comment:
couldn't we imply add a folder with preconverted png-files and enhance this rule that it checks for existence of $(INKSCAPE_CMD) and if it doesn't exist it simply copies the png-file from the preconverted directory. Everything else could stay as is and we could simply control it via having inkscape installed in a build-sandbox or not. We could have %bcond_with png_generation for that to make it more comfortable.
So nobody would ever have to rebuild pngs manually. For convenience we might add a make target to get generated pngs copied over prebuild ones.

Reply:
Not worth it, those files never change. If we ever update them, it will likely be a one-time thing.

Reply:
Yep - at this point you are right ;-)
Before investing the effort you did I'd think it would have some about equal.
Anyway it is done and does the job and we'll probably never touch it.